spp_change_request/security/ir.model.access.csv

```diff
@@ -34,6 +34,10 @@ spp_ir_model_validator,SPP IR Model Validator Access,base.model_ir_model,spp_cha
 spp_ir_model_fields_validator,SPP IR Model Fields Validator Access,base.model_ir_model_fields,spp_change_request.group_spp_change_request_validator,1,0,0,0
 spp_ir_sequence_validator,SPP IR Sequence Validator Access,base.model_ir_sequence,spp_change_request.group_spp_change_request_validator,1,0,0,0
 
+spp_idqueue_validator,SPP ID Queue Validator Access,spp_idqueue.model_spp_print_queue_id,spp_change_request.group_spp_change_request_validator,1,1,1,0
+spp_entitlement_finance_validator,SPP Entitlement Finance Validator Access,g2p_programs.model_g2p_entitlement,spp_change_request.group_spp_change_request_validator,1,1,1,0
+spp_program_membership_program_validator,SPP Program Membership Program Validator Access,g2p_programs.model_g2p_program_membership,spp_change_request.group_spp_change_request_validator,1,1,1,0
```

Comment:
Remove all with `g2p_programs.model_*`. We cannot add as dependency the g2p_programs in the change request module. I think you do not need the g2p_programs models here.